package/.storybook/preview.ts

```diff
@@ -1,10 +1,11 @@
 import './preview.css';
 
-import { withConsole } from '@storybook/addon-console';
+// Known issue here: https://github.com/storybookjs/storybook-addon-console/issues/80
+// import { withConsole } from '@storybook/addon-console';
```

Comment:
Can enable this feature back when https://github.com/storybookjs/storybook-addon-console/issues/80 resolved.

Without that, the console.log won't print to Action tab.